Resources/Prototypes/_L5/Entities/Structures/Holographic/projections.yml

```diff
@@ -0,0 +1,69 @@
+- type: entity
+  id: StaticFieldMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: static field projector machine board
+  description: A machine printed circuit board for a static field projector.
+  components:
+  - type: MachineBoard
+    prototype: StaticField
+    stackRequirements:
+      Gold: 2
+      Cable: 4
+      Glass: 2
+
+- type: entity
+  id: StaticField
+  parent: [ BaseMachinePowered, ConstructibleMachine ]
+  name: static field projector
+  description: A powered barrier of hard light that blocks air, but nothing else.
```

Comment:
Probably something-something-polarization; hardlight is magic and I wanted to aim for the *aesthetic* of hard scifi. Remember the static field on La Bénção? Same thing.

Reply:
Switched to high-efficiency teslaphoresis, which would also make the hum make sense. https://en.wikipedia.org/wiki/Tesla_coil#Teslaphoresis

Reply:
Based as fuck, thanks for that